docs/deep-dives/pnpm.md

```diff
@@ -279,7 +281,7 @@ This migration guide assumes that your project was scaffolded with a **skuba** t
     Your build pipeline may have previously output an ephemeral `.npmrc` with an auth token on the build agent.
     This needs to be output elsewhere to avoid overwriting the new pnpm configuration stored in `.npmrc`.
 
-    Swap out caching on `package.json` and `yarn.lock` for `.npmrc` and `pnpm-lock.yaml` at the same time.
+    Swap out caching on `yarn.lock` for `.npmrc` and `pnpm-lock.yaml` at the same time.
```

Comment:
cc @samchungy as I know you have opinions about caching

Reply:
Sadge but there is no better solution imo